src/pages/preview.vue

```diff
@@ -187,13 +239,63 @@ export default defineComponent({
     const displayLog = () => {
       console.log("click");
     };
-    // WelcomeModal
-    const welcomeModal = ref(false);
-    const openWelcomeModal = () => {
-      welcomeModal.value = true;
+
+    // popup
+    const popupClickHandler = (value: string) => {
+      console.log(value);
+    };
+
+    const popupData = [
```

Comment:
popupcontent側で

`type Color = "normal" | "danger";`

を定義して

```suggestion
    const popupData: {
      value: string;
      link: boolean;
      onClick: Function;
      color: Color;
    }[] = [
```

とかしてほしい:pray:

Reply:
popupDataの型はpreviewとpopupcontentのどちらに書けばいいですか？

Reply:
popupcontentで使わないので、previewだと思う。
popupDataを複数のページで使うようになったら、その時に移動しようかなー